executor/common_linux.h

```diff
@@ -5574,3 +5577,43 @@ static long syz_pkey_set(volatile long pkey, volatile long val)
 	return 0;
 }
 #endif
+
+#if SYZ_EXECUTOR || SYZ_SWAP
+#include <fcntl.h>
+#include <stdio.h>
+#include <string.h>
+#include <sys/stat.h>
+#include <sys/swap.h>
+#include <sys/types.h>
+
+#define SWAP_FILE "./swap-file"
+#define SWAP_FILE_SIZE (128 * 1000 * 1000) // 128 MB.
+
+static void setup_swap()
+{
+	// The call must be idempotent, so first disable swap and remove the swap file.
+	swapoff(SWAP_FILE);
+	unlink(SWAP_FILE);
+	// Zero-fill the file.
+	int fd = open(SWAP_FILE, O_CREAT | O_WRONLY | O_CLOEXEC, 0600);
+	if (fd == -1) {
+		failmsg("open failed", "file: %s, errno: %d\n", SWAP_FILE, errno);
```

Comment:
failmsg always prints errno, no need to print it

Reply:
Fixed